man/ostree.repo-config.xml

```diff
@@ -181,6 +181,14 @@ Boston, MA 02111-1307, USA.
       </varlistentry>
 
       <varlistentry>
+        <term><varname>require-deltas=upgrade</varname></term>
```

Comment:
Are you planning for this to be a list of scenarios in the future? Would be cleaner & clearer to make it a boolean, e.g. `require-delta-upgrades` or `require-deltas-on-upgrade`, no?